### PR TITLE
Create aims-and-values.md

### PR DESCRIPTION
**Pull request by SaraJaoude**
closes #ISSUE-NO

### CODE REVIEW

File: docs/aims-and-values.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# EddieHub Aims and Values
+
+## Our Background
+ 
+EddieHub was founded by Eddie Jaoude, GitHub Star, GitHub Star of the Year 2020 and winner of the GitHub Community Growth Award 2021. As a passionate advocate of Open Source who has always believed in giving back to the tech community, Eddie’s particular focus is creating and supporting opportunities for those wanting to start a career in tech.
+ 
+Recognising that starting in Open Source can be daunting, Eddie was keen to build a community which focused on Open Source for those with varying tech abilities and at different stages of their tech journey.
+ 
+The main objective was for members to share their experiences by adopting both a mentor and mentee role on the premise that everyone has something to teach but also something to learn. “Collaboration First, Code Second” quickly became what the community members live by.
+
```
Comment (schmelto) [inline docs/aims-and-values.md:10]:
```suggestion
```

File: docs/aims-and-values.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ 
+By creating a community with members from all around the world and with different technical levels, EddieHub has generated an opportunity for members to interact with a large diversity of people. This constant interaction between members means that the way they receive feedback has an impact on how they communicate feedback to others.
+ 
+Recognising that at the outset these communication skills were primarily through the Discord written messages and GitHub Issues and Pull Requests, EddieHub has launched initiatives for regular Public Speaking events, tutorials and “Ask me Anything” sessions to further support its members.
+ 
+We are pleased that members have reported that this approach has been invaluable to them. Had they not made their projects Open Source, they would not have been able to progress these in the same way as a private project. In addition, whilst feedback was either not sought or viewed as a negative critique, it is now both actively encouraged and more confidently given.
+
```
Comment (schmelto) [inline docs/aims-and-values.md:35]:
```suggestion
```

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Comment by schmelto** [comment]
Maybe add a link to the sidebar docs/_layout/sidebar.md

**Comment by stemount** [comment]
@schmelto the last line e.g. EOF is in a newline as this has been the standard for a number of languages.

**Pull request merged**